SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealObjectRef.h

```diff
@@ -72,7 +72,11 @@ struct SPATIALGDK_API FUnrealObjectRef
 	}
 
 	static UObject* ToObjectPtr(const FUnrealObjectRef& ObjectRef, USpatialPackageMapClient* PackageMap, bool& bOutUnresolved);
```

Comment:
It seems these `static` converter functions are straightforward to unit-test? :)

Reply:
Hmm I can think about it, but resolving stably named objects need a project with some assets to be able to test it.
Is there a test which sets up the required environment and which I could start with ?

Reply:
As far as I know - we use TestGym now, so maybe adding some asset there and testing against it would work? But if it's more difficult then that - I will not block the PR on it.

Reply:
--> https://github.com/spatialos/UnrealGDKTestGyms/pull/18

Reply:
I thought of unit-tests, but that's also good :)